backend/baseclass.pm

```diff
@@ -427,7 +427,7 @@ sub can_handle ($self, @) {
     return;    # sorry, no
 }
 
-sub do_extract_assets ($self) { notimplemented }
+sub do_extract_assets ($self, $@) { notimplemented }
```

Comment:
```suggestion
sub do_extract_assets ($self, $args) { notimplemented }
```

Reply:
We don't give names to arguments in stubs currently - but if you don't mind inconsistency I'm happy to change it

Reply:
c.f.

    backend/null.pm
    sub do_extract_assets ($self, @) { }

Reply:
Well, `$@` was a syntax error. And using `@` says it can have more than two arguments, which is wrong. So if you don't want to give it a name, use `$`.